### DISCUSSION TITLE
Support ApacheDoris

### DISCUSSION THREAD

**Discussion by Jengro777** [rudderlabs/rudder-server#3666] [Feature requests]
Support ApacheDoris

**Comment by gitcommitshow** [comment]
Thank you for submitting the integration request. Will look into this.
Feel free to share more context that can help the team understand requirements in more detail (is it source or destination, what is the use case, any specific feature you have in mind, etc.)

**Reply by gitcommitshow** [reply]
Hi @Avey777 , can you help with more context to prioritise this integration
